src/renderer/components/Editor/Editor.tsx

```diff
@@ -36,9 +46,9 @@ export default class Editor extends React.Component<any, any> {
       [process.platform === "darwin" ? "Cmd-/" : "Ctrl-/"]: () => {
         this.codeMirror.execCommand("toggleComment");
       },
-      Tab: cm => {
+      Tab: (cm: CodeMirror.Editor) => {
         if (!cm.state.vim) {
-          if (cm.somethingSelected()) cm.indentSelection("add");
+          if (cm.getDoc().somethingSelected()) cm.execCommand("indentMore");
```

Comment:
`indentSelection` is not public function.
https://github.com/codemirror/CodeMirror/pull/6083#issuecomment-563124716